tests/unit-tests/test_gnn.py

```diff
@@ -122,7 +122,7 @@ def require_cache_embed(self):
     pred1, labels1 = node_mini_batch_predict(model, embs, dataloader1, return_label=True)
     dataloader2 = GSgnnNodeDataLoader(data, target_nidx, fanout=[-1, -1],
                                       batch_size=10, device="cuda:0", train_task=False)
-    pred2, emb2, labels2 = node_mini_batch_gnn_predict(model, dataloader2, return_label=True)
+    pred2, _, labels2 = node_mini_batch_gnn_predict(model, dataloader2, return_label=True)
```

Comment:
Do we also need to add edge_mini_batch_gnn_predict() for testing.

Reply:
We do. In L263